### ISSUE TITLE
BlankState does not create elasticsearch index

### ISSUE THREAD

**Issue by konzz** [huridocs/uwazi#2585]
On a fresh Uwazi install, where the elasticsearch index does not exists, runing  yarn blank-state, does not create the index, and when you open the app it shows nothing, an empty page, but also does not throw any errors.

**Comment by txau** [comment]
Apparently it throws an error when trying to delete:
```
Deleting index... uwazi_development
{ json:
   { error:
      { root_cause: [Array],
        type: 'index_not_found_exception',
        reason: 'no such index [uwazi_development]',
        'resource.type': 'index_or_alias',
        'resource.id': 'uwazi_development',
        index_uuid: '_na_',
        index: 'uwazi_development' },
     status: 404 },
  status: 404,
  cookie: undefined }
Creating index... uwazi_development
```

**Comment by txau** [comment]
@konzz the error above mentioned does not prevent the creation of the new index. It worked for me in a fresh install. Maybe the error/blank screen was due to another problem? 

I'm going to close this issue unless you confirm otherwise.

**Comment by konzz** [comment]
Looks like it was fixed here #2615
